tests/core/actions/test_forms.py

```diff
@@ -446,33 +446,31 @@ async def test_validate_slots_on_activation_with_other_action_after_user_utteran
     ]
 
 
-def test_name_of_utterance():
-    form_name = "another_form"
+@pytest.mark.parametrize(
+    "utterance_name, expected",
+    [
+        ("utter_ask_my_form_num_people", "utter_ask_my_form_num_people"),
```

Comment:
if these are the same, why do we need to parameters?

Reply:
@wochinge could you please take this over